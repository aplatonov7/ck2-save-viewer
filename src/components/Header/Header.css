.Header {
  padding: 1rem;
  background-color: #3f51b5;
  color: #fff;
  box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
  z-index: 5;
}