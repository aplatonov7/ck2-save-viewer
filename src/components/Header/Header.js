import React from 'react'

import './Header.css'

const Header = () => (
  <header className="Header">
    CK2 save to stats
  </header>
)

export default Header
