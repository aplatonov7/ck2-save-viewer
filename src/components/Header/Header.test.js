import React from 'react'
import Header from './Header'

describe('Header component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
