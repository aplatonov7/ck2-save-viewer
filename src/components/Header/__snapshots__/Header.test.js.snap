// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<header
  className="Header"
>
  CK2 save to stats
</header>
`;
